src/Hypergrid/scrolling.ts

```diff
@@ -201,6 +201,7 @@ exports.mixin = {
             this.behavior.changed();
             var oldX = this.hScrollValue;
             this.hScrollValue = x;
+            this.sbHScroller.index = x;
```

Comment:
This is required to update the value in the scroller so the grid doesnt randomly jump back to the old position after a shape change